openlibrary/catalog/works/find_works.py

```diff
@@ -750,8 +745,6 @@ def other_matches(w, existing_wkey):
     works_updated_this_session = set()
 
     for w in works: # 4th pass
-        if 'has_conflict' in w:
-            pprint(w)
         assert 'has_conflict' not in w
```

Comment:
Help the developer to understand failures by adding a string to the end of the assert on line 748
* assert ‘has_conflict’ not in w, ‘w: {}’.format(w)